api/lib/application/organizations/index.js

```diff
@@ -173,8 +173,8 @@ exports.register = async (server) => {
           }),
         },
         payload: {
-          maxBytes: 1048576 * 10, // 10MB
-          parse: 'gunzip',
+          maxBytes: 1048576 * 20, // 20MB
```

Comment:
Je propose de créer une variable d'environnement, cela permettra de s'adapter si besoin

Comme noté dans la PR #2109, un fichier SIECLE de 4000 élèves fait au moins 10 Mo (sans balise OPTION)
La plus grande organisation compte déjà 3640 membres

```
  id   | count 
-------+-------
  1562 |  3640
  5370 |  3581
  5592 |  3515

SELECT
     o.id, COUNT(1)
FROM
    "schooling-registrations" sr
         INNER JOIN organizations o on sr."organizationId" = o.id
GROUP BY o.id
ORDER BY COUNT(1) DESC
;
```

Reply:
Effectivement, une variable d'env avec une valeur par défaut peut éviter un déploiement si on veut mettre à jour sans déploiement en cas de modification.